### ISSUE TITLE
🐛 BUG: Language dropdown doesn't open by keyboard when Narrator is on

### ISSUE THREAD

**Issue by RafidMuhymin** [dilmaheu/dilmahtea.me#245]
**Describe the bug**
The language selectors dropdown doesn't open with the `Enter` key  when Narrator is on.

**Steps to Reproduce**

1. Go to https://dilmahtea.me/ with Narrator (Windows) turned on
2. See that the language dropdown can't be opened by keyboard when Narrator is on.

**Expected behavior**
It should be possible to open the language selectors dropdown by keyboard with/without Narrator turned on.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Version: 110

**Additional context**
N/A
